packages/inventory-compliance/src/ComplianceEmptyState/ComplianceEmptyState.js

```diff
@@ -68,7 +68,7 @@ const ComplianceEmptyState = ({ title, mainButton, client }) => {
             <EmptyStateSecondaryActions>
                 <Button variant='link' component='a' target='_blank' rel='noopener noreferrer'
                     href={ `https://access.redhat.com/documentation/en-us/red_hat_insights/` +
-                           `2020-04/html/assessing_and_monitoring_security_policy_compliance_of_rhel_systems/index` } >
+                           `2021/html/assessing_and_monitoring_security_policy_compliance_of_rhel_systems/index` } >
```

Comment:
Probably we can move this to `constants.js`?

Reply:
Should I create that file? In this context it doesn't exist...

Reply:
Oh, I haven't realized that this is the frontend components repo... There is `src/Constants/Constants.js` within the package.